deps/reftable/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+disable_warnings(sign-compare)
+disable_warnings(declaration-after-statement)
+disable_warnings(unused-parameter)
```

Comment:
I don't care too much about sign-comparison and unused parameter warnings, even though it'd be nice to not have to disable them. I feel like the declaration-after-statement warnings got a point, though, to make it C90 compatible like libgit2 aims to be.

Reply:
the main code is c90 clean, but I guess you're compiling tests too?

Reply:
Initially I did, but I've removed them as soon as I noticed. So maybe it's not necessary anymore to disable those warnings, let me check again.

Reply:
I've checked again, it's "dump.c". Didn't look like an executable that's not part of the library, which is why I've compiled it in. Maybe it would be worthwhile to split out tests as well as standalone executables into their own subdirectory to allow just adding "*.c" to your project.